DataFormats/HcalDetId/interface/HcalDetId.h

```diff
@@ -38,6 +38,8 @@ class HcalDetId : public DetId {
   int iphi() const { return id_&0x7F; }
   /// get the tower depth
   int depth() const { return (id_>>14)&0x1F; }
+  /// get the tower depth
+  uint32_t maskDepth() const { return (id_ | 0x1C000); }
```

Comment:
Sounds like a good idea to me, especially if some of the masks are related and one can just write down an expression showing the relationship as part of the ```static constexpr```. That way it also works as documentation.